### PR TITLE
Rate page

### PR DESCRIPTION
**Pull request by AmrNabel**

### CODE REVIEW

File: src/containers/Search/index.tsx [hunk 1]
```diff
@@ -133,6 +134,7 @@ const Search = (homeProps: IHomeProps) => {
     event.preventDefault()
     const claim = event.target
     const currentClaim = claim.data('raw')
+    console.log('plaaaaaaaay', currentClaim)
```
Comment (youngnishant) [inline src/containers/Search/index.tsx:137]: Remove console log before pushing code

File: src/App.tsx [hunk 2]
```diff
@@ -11,6 +11,8 @@ import './App.css'
 import { CssBaseline, ThemeProvider, createTheme } from '@mui/material'
 import Box from '@mui/material/Box'
 import FeedClaim from './containers/feedOfClaim/index'
+import Rate from './components/Rate'
+import { BrowserRouter as Router } from 'react-router-dom'
```
Comment (youngnishant) [inline src/App.tsx:15]: Library imports should be listed before local components or other imports

File: src/App.tsx [hunk 3]
```diff
@@ -96,8 +98,14 @@ const App = () => {
             <Route path='feed' element={<FeedClaim {...commonProps} />} />
             <Route path='search' element={<Search {...commonProps} />} />
             <Route path='/' element={<Form {...commonProps} />} />
-            <Route path='login' element={<Login {...commonProps} />} />
             <Route path='register' element={<Register {...commonProps} />} />
+            <Route path='login' element={<Login {...commonProps} />} />
+            <Route
+              path='/rate'
```
Comment (youngnishant) [inline src/App.tsx:104]: Make a config array and map it into route

File: src/components/Rate/index.tsx [hunk 4]
```diff
@@ -0,0 +1,212 @@
+import Box from '@mui/material/Box'
+import { useNavigate, useParams } from 'react-router-dom'
+import Typography from '@mui/material/Typography'
+import { TextField, Button, FormControl, MenuItem, Rating, FormHelperText } from '@mui/material'
+import IHomeProps from '../../containers/Form/types'
+import styles from '../../containers/Form/styles'
+import { Controller, useForm } from 'react-hook-form'
+import { useCreateClaim } from '../../hooks/useCreateClaim'
+import Tooltip from '@mui/material/Tooltip'
```
Comment (youngnishant) [inline src/components/Rate/index.tsx:9]: Same list it before local imports

File: src/App.tsx [hunk 5]
```diff
@@ -96,8 +98,14 @@ const App = () => {
...
+              element={
+                checkAuth() ? <Rate {...commonProps} /> : <Navigate to='/login' replace state={{ from: location }} />
```
Comment (youngnishant) [inline src/App.tsx:106]: Make a common function that is reusable

**Review by gvelez17 — APPROVED** [review]

File: src/App.tsx (hunk 3 above)
Comment (gvelez17) [reply src/App.tsx:104]: hey @sahdasamier @Zakiamangal you guys could do this suggestion later?

File: src/App.tsx (hunk 5 above)
Comment (gvelez17) [reply src/App.tsx:106]: this is also good idea!  @sahdasamier @Zakiamangal if you want to take this

File: src/components/Rate/index.tsx (hunk 4 above)
Comment (gvelez17) [reply src/components/Rate/index.tsx:9]: @AmrNabel this is good idea!  please do in future PR

**Pull request merged**